pages/index.js

```diff
@@ -1,5 +1,138 @@
-const Home = () => {
-  return <h1>WELCOME :D</h1>;
+import { useState, useRef } from 'react';
+import Image from 'next/image';
+import PropTypes from 'prop-types';
+import axios from 'axios';
+import { getSession } from 'next-auth/client';
+import Button from '@material-ui/core/Button';
+import TextField from '@material-ui/core/TextField';
+
+import User from '../models/User';
+import dbConnect from '../utils/dbConnect';
+
+const Home = ({ session, acceptedFileTypes, allowMultipleFiles }) => {
+  session = session !== '' && JSON.parse(session);
```

Comment:
maybe u should pass in `session` as the actual object instead of accepting string and parsing it. Other people kind of have to guess that they have to pass a string that just so happens to be a json object

Reply:
Ahaha that would be the ideal way of doing it but unfortunately next complains. Look at comment in line 135